CHANGELOG.txt

```diff
@@ -1,4 +1,5 @@
 2.0.1-SNAPSHOT
+* Remove build dependency to GPG for automation purposes
```

Comment:
Shouldn't this be as part of the 2.0.0 release? We haven't released that version yet.

Reply:
I thought about force pushing this to master but I felt a little extreme. What do you think?

Reply:
Humm... I guess I can push this as 2.0.0 and just create a new release PR with the same version number.

Reply:
I guess just create a new release PR that backs out the SNAPSHOT version and then readds it, so we have a clean commit to release.

Reply:
I think we can merge this as it is and I'll send another release PR reformating the changelog. I think it will be more clear that way...